frontend/src/utils/generate-warnings.ts

```diff
@@ -881,16 +890,25 @@ function checkSemesterCredits(
     minCredits += course.numCreditsMin; //use this one!
   }
 
-  //minSeasonCredits is the minimum number of credits to not be under enrolled for the given season.
-  let minSeasonCredits = seasonCreditTracker[season].seasonMin;
-
-  //maxSeasonCredits is the maximum number of credits to not be over enrolled for the given season.
-  let maxSeasonCredits = seasonCreditTracker[season].seasonMax;
-
   //defining a list of IWarnings to return later.
   const warnings: IWarning[] = [];
 
-  //if currently planning to few credits for the given season (needs at least one credit to throw warning).
+  let minSeasonCredits;
+  let maxSeasonCredits;
+
+  //taking classes normally
+  if (status == "CLASSES") {
+    //minSeasonCredits is the minimum number of credits to not be under enrolled for the given season.
+    minSeasonCredits = seasonCreditTracker[season].seasonMin;
+
+    //maxSeasonCredits is the maximum number of credits to not be over enrolled for the given season.
+    maxSeasonCredits = seasonCreditTracker[season].seasonMax;
+  } else {
```

Comment:
Switch this to check if the status is coop, since right now this is including vacation semesters, and have it return no warnings if it's a vacation block